tests/GuardTest.php

```diff
@@ -24,26 +25,28 @@
 
 class GuardTest extends TestCase
 {
-    /**
-     * @expectedException Exception
-     * @expectedExceptionMessage CSRF middleware instantiation failed. Minimum strength is 16.
-     */
+    use ProphecyTrait;
+
     public function testStrengthLowerThan16ThrowsException()
     {
         $storage = [];
         $responseFactoryProphecy = $this->prophesize(ResponseFactoryInterface::class);
-        $mw = new Guard($responseFactoryProphecy->reveal(), 'test', $storage, null, 200, 15);
+
+        $this->expectException(Exception::class);
+        $this->expectExceptionMessage('CSRF middleware instantiation failed. Minimum strength is 16.');
```

Comment:
nitpick: can we put the expectException statements at the very beginning of the function declaration? It reads easier in my opinion

```php
public function test(): void
{
    // expect
    // expect
   
    // logic
}
```

Reply:
I'm on the fence about this. Best practice is for the `expectException()` to be immediately before the call that will throw so that if a previous line of code in the test happens to throw, the test doesn't pass by mistake.

Reply:
Good point!